### PR TITLE
Styling for Feature/9508 site search

### PR DESCRIPTION
**Pull request by willhall88**
This is the main styling and layout changes to supplement the great work done by @MarcoBgn on adding in the site search feature to fincap  [TP9508](https://moneyadviceservice.tpondemand.com/entity/9508-add-site-search-to-fincap)

@davidtrussler @rhianwatkins1, please pick up any changes in my absence and apply them (hopefully it's in a nice enough state that there's nothing too bad!)

### CODE REVIEW

**Comment by willhall88** [comment]
This PR points to another feature branch. Most of the changes are currently up on UAT for testing, I made a few tweaks after deploying that, but nothing in terms of the look and functionality you should see on UAT. 

if and when this PR get's approved it will need to go into the current open PR #190  and then both can get merged into Master branch from there :)

File: app/assets/stylesheets/components/ui/_search_results.scss [hunk 1]
```diff
@@ -25,3 +25,19 @@
 .search-results__evidence-type {
   margin: $baseline-unit*3 0;
 }
+
+input.search__input--inpage {
```
Comment (davidtrussler) [inline app/assets/stylesheets/components/ui/_search_results.scss]: Shouldn't need to specify input with BEM notation.

File: app/assets/stylesheets/base/_forms.scss [hunk 2]
```diff
@@ -40,3 +40,8 @@ label input[type=checkbox] {
   margin: 0 $baseline-unit;
 }
 
+.inpage {
```
[the remark this answers is not in the document]
Comment (willhall88) [reply app/assets/stylesheets/base/_forms.scss]: I think I missed taking this styling out, when I was cleaning up. apologies

**Comment by willhall88** [comment]
Thanks for the comments. A lot of this work was being done at 5pm on Friday before I went on holiday so definitely was rushed work. If I get a chance I will look to clean up some of the above comments in a separate PR as some of them are well outside of our standards

**Pull request merged**